pkg/workflows/create.go

```diff
@@ -247,13 +249,30 @@ func (s *MoveClusterManagementTask) Run(ctx context.Context, commandContext *tas
 		return &CollectDiagnosticsTask{}
 	}
 
-	return &InstallEksaComponentsTask{}
+	return &InstallEksdComponentsTask{}
 }
 
 func (s *MoveClusterManagementTask) Name() string {
 	return "capi-management-move"
 }
 
+// InstallEksdComponentsTask implementation
+
+func (s *InstallEksdComponentsTask) Run(ctx context.Context, commandContext *task.CommandContext) task.Task {
```

Comment:
Does this need to be a new task?

In fact, can this just be part of install eksa components in cluster manager? This not different than installing the bundle